javascript-conditionals/main.js

```diff
@@ -0,0 +1,113 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  return number < 5;
+}
+console.log('isUnderFive(2):', isUnderFive(2));
+console.log('isUnderFive(6):', isUnderFive(6));
+
+function isEven(number) {
+  if (number % 2) {
+    return false;
+  }
+  return true;
+}
+console.log('isEven(1):', isEven(1));
+console.log('isEven(2)', isEven(2));
+
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  }
+  return false;
+}
+console.log("startsWithJ('Tim'):", startsWithJ('Tim'));
+console.log("startsWithJ('JavaScript'):", startsWithJ('JavaScript'));
+
+const person1 = { name: 'Tim', age: 21 };
+const person2 = { name: 'Joe', age: 13 };
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+console.log('isOldEnoughToDrink of 21 year old:', isOldEnoughToDrink(person1));
+console.log('isOldEnoughToDrink of 13 year old:', isOldEnoughToDrink(person2));
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+console.log('isOldEnoughToDrive of 21 year old:', isOldEnoughToDrive(person1));
+console.log('isOldEnoughToDrive of 13 year old:', isOldEnoughToDrive(person2));
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+console.log(
+  'isOldEnoughToDrinkAndDrive of 21 year old:',
+  isOldEnoughToDrinkAndDrive(person1)
+);
+console.log(
+  'isOldEnoughToDrinkAndDrive of 13 year old:',
+  isOldEnoughToDrinkAndDrive(person2)
+);
+
+function categorizeAcidity(pH) {
+  if (pH < 7 && pH >= 0) {
+    return 'acid';
+  } else if (pH > 7 && pH < 14) {
```

Comment:
14 would still be considered a base, so this should be less than or equal to

Reply:
![Screen Shot 2023-05-10 at 1 54 05 PM](https://github.com/silva-tim/c0523-code-solutions/assets/114685872/21e457cc-f073-4769-b5a1-619a873478e9)


Fixed